lab3.c

```diff
@@ -0,0 +1,21 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+int search (int a[],int n) {
```

Comment:
replace a and n, because it is not clear
```suggestion
int search (int input_arr[],int once_num) {
```